test/integration/tasks.test.js

```diff
@@ -290,6 +290,23 @@ describe("Tasks", function () {
           return done();
         });
     });
+    it("Should update the task status collapsed for the given taskid", function (done) {
+      chai
+        .request(app)
+        .patch("/tasks/" + taskId1)
+        .set("cookie", `${cookieName}=${jwt}`)
+        .send({
+          isCollapsed: true,
+        })
+        .end((err, res) => {
+          if (err) {
+            return done(err);
+          }
+          expect(res).to.have.status(204);
+
+          return done();
```

Comment:
does the response object also returns the modified data. if yes then please test the isCollapsed from the response.

Reply:
It is optional

Reply:
test added in last commit @heyrandhir